src/RpcClient/RpcClient.cs

```diff
@@ -199,34 +210,34 @@ public RpcRawMemPool GetRawMempoolBoth()
         /// <summary>
         /// Returns the corresponding transaction information, based on the specified hash value.
         /// </summary>
-        public string GetRawTransactionHex(string txid)
+        public string GetRawTransactionHex(string txHash)
         {
-            return RpcSend("getrawtransaction", txid).AsString();
+            return RpcSend("getrawtransaction", txHash).AsString();
         }
 
         /// <summary>
         /// Returns the corresponding transaction information, based on the specified hash value.
         /// verbose = true
         /// </summary>
-        public RpcTransaction GetRawTransaction(string txid)
+        public RpcTransaction GetRawTransaction(string txHash)
         {
-            return RpcTransaction.FromJson(RpcSend("getrawtransaction", txid, true));
+            return RpcTransaction.FromJson(RpcSend("getrawtransaction", txHash, true));
         }
 
         /// <summary>
         /// Returns the stored value, according to the contract script hash and the stored key.
         /// </summary>
-        public string GetStorage(string script_hash, string key)
+        public string GetStorage(string scriptHash, string key)
```

Comment:
Maybe this param name could be changed? Since we can also use `Id` to get storage.

Reply:
We should accept both, because it's easier for the user know the script hash.

Reply:
Yes, I have changed the code, please have a check.